TestAssemblies/SimpleTest/AsyncClass.cs

```diff
@@ -5,17 +5,17 @@ namespace SimpleTest {
     public class AsyncClass {
         [Interceptor]
         public async Task SimpleAsyncMethod() {
-            await Task.Delay(1);
+            await Task.Delay(1).ConfigureAwait(false);
```

Comment:
I don't think we do care about threads at all in this test. All that we want is just check how correct IL was rewritten and check that both part of async function was rewritten. I don't think this change is necessary.